pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -1,37 +1,63 @@
+{{ if .DecryptedEnvelopes }}
+{{ range .DecryptedEnvelopes }}
```

Comment:
The transaction detail info listed for the secure envelopes, to match Figma, is in the `DecryptedEnvelope` so this is why the `SecureEnvelope` was replaced here.